experimental/dds/tree2/src/feature-libraries/chunked-forest/codec/fence.json

```diff
@@ -0,0 +1,5 @@
+{
+	"tags": ["chunked-forest-codec"],
```

Comment:
as we are migrating away from using good-fences, I don't think adding this has much value. Also limiting what this codec and depend on vs its parent folder can depend on isn't too useful when both are depending on each-other so this doesn't limit transitive deps at all.

Reply:
removed tag

Reply:
this comment was intended to apply to this while file.

Reply:
removed fence.json file